### DISCUSSION TITLE
Disconnected from manager

### DISCUSSION THREAD

**Discussion by 011248163264** [Security-Onion-Solutions/securityonion#8039] [Unsupported Versions]
After installation, while reviewing the freshly-installed Security Onion console, I see a message at the top. It toggles between "Attempting to connect to manager" and "Disconnected from manager" continuously, every few seconds. It's a standalone install.. So-status shows that everything it sees is OK. Where should I look to begin resolving this?

the server have only one static lokal ipadress with no proxy i connect direct to the lokal ip with ssl.
The ssl is a selfsign ssl cert.

when i restart the server is it working a few minuts

version 2.3.90

**Comment by n0th1nt0lOOz** [comment]
Same problem in 2.3.120. Web interface show status "Disconnected from manager" and "Attempting to connect to manager" too
All port are opened
I use a nginx reverse proxy nginx with company's certificate and forward to web

**Reply by tiger-greenhell** [reply]
Attention. It is necessary to think that part of the communication is realized via websocket.

**Reply by dougburks** [reply]
@tientmse62290 Please see @tiger-greenhell's response and, if you have further questions or problems, please start a new discussion. From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

**Comment by dougburks** [comment]
@01123581321345589144233 What browser are you using?

Have you checked the logs in `/opt/so/log/soc/` for additional clues?

**Reply by 011248163264** [reply]
System are Centos7 standalone version 2.3.120 4core with 32Gib Ram lokal storage, all ssd drives
1 node with 4 wazuh agents, trafik line is a 60Mib up and download
so-status shows [OK]
Gridpage shows Status OK

**Reply by 011248163264** [reply]
/opt/so/log/soc/sensoroni-server.log
4139 timestamp=2022-06-01T01:10:11.759056265Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:47968 requestId=48d6d921-2b97-4c74-a62d-d72b2fb61f1b requestor=agent sourceIp=192.168.1.95 statusCode=200

**Reply by 011248163264** [reply]
Browser are Firefox Safari

**Reply by dougburks** [reply]
Have you tried a chromium-based browser?

From https://docs.securityonion.net/en/2.3/soc.html:
> We recommend chromium or chromium-based browsers such as Google Chrome. Other browsers may work, but fully updated chromium-based browsers provide the best compatibility.

**Reply by 011248163264** [reply]
yes its the same

**Reply by dougburks** [reply]
Are you able to duplicate this issue on a fresh installation (perhaps in a VM)?

**Reply by najiyousif** [reply]
Hi Guys,
same problem here, it was working fine for a long time, now "disconnected from manager" error comes up all day long, log file showing nothing wrong.

**Reply by dougburks** [reply]
@najiyousif From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.

**Reply by 011248163264** [reply]
i have reinstalled sos and afte a while is the same issue: Disconnectet from manager, the same issue was also in the version before

**Reply by 011248163264** [reply]
you can't assume that everyone wants to work with chromium browser.
It's a websocket connection and that must work in other browsers too.
or the connection must be made differently.

**Reply by 011248163264** [reply]
but there is a general issue with websockets in different browsers

**Reply by 011248163264** [reply]
update: i have closed all my sos windows and tabs, and login again and there is no websocket disconnectet!

**Reply by 011248163264** [reply]
update: after 3h with grafana and osquery open the sos have a ws timeout again, i close the windows and tabs and the timeout i gone

**Reply by dougburks** [reply]
Have you tried adjusting the session timeout value?
https://docs.securityonion.net/en/2.3/soc-customization.html#session-timeout

**Reply by 011248163264** [reply]
i have set this to 720h, so i try :-)

**Reply by 011248163264** [reply]
update: ws disconnectet, but when i close the windows again, grafana , kibanna..... after 2min the disconnect is gone.

**Reply by dougburks** [reply]
Is it possible that the machine that your browser is running on has minimal RAM and having too many tabs open is causing problems?

**Reply by 011248163264** [reply]
Hi i have 16GB ram yes and 10-15 tabs open, but they are not sos tabs. i have updatet to 2.3.130 and
the issue is still there. The issue is only when i open more sos tabs, Grafana, Kibana ......

**Reply by dougburks** [reply]
This is a strange issue. Do you have Wazuh enabled and, if so, have you checked to see if perhaps Wazuh Active Response could be interfering?

**Reply by 011248163264** [reply]
yes wazuh is enabled on all 9 hosts
"Do you have Wazuh enabled and, if so, have you checked to see if perhaps Wazuh Active Response could be interfering?"
i try to figure out

**Reply by Njinx** [reply]
@01123581321345589144233 Did you ever figure this out?
